README.md

```diff
@@ -457,6 +458,13 @@ This sample code contains implementations of the models given in the
                              average-pooling on the fly over frame-level
                              features rather than using pre-aggregated features.
 
+## Create Your Own Dataset Files
+You can create your dataset files from your own videos. Our
+[feature extractor](./blob/master/feature_extractor) code creates `tfrecord`
```

Comment:
This link doesn't work for me.

Reply:
It will work, once this is checked in, I think